coralproject/talk
# [CORL-2746] Allow admins to demote other admins

## What does this PR do?

Allows admin users to demote other admin users in the `Admin > Community` tab.

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags? 

No

## If any indexes were added, were they added to `INDEXES.md`?

No new indices.

## How do I test this PR?

- sign in as admin
- promote someone to admin
- refresh page and see the promotion stuck
- demote them from admin to something else
- refresh page and see the demotion stuck 
- sign in as org mod, site mod, and see that you cannot demote admins

## How do we deploy this PR?

No special considerations.
